src/content/docs/errors-inbox/apm-tab.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: 'APM: Group errors tab '
+tags:
+  - APM
+  - APM UI pages
+  - Error analytics
+metaDescription: New Relic's error profiles feature in the classic APM view shows trends and anomalies that help you compare and troubleshoot error events.
+redirects:
```

Comment:
do we have redirects somewhere for the other deleted docs?

Reply:
yes! I added redirects to this doc: src/content/docs/apm/apm-ui-pages/error-analytics/errors-page-find-fix-verify-problems.mdx